tests/tests/image_atomics/mod.rs

```diff
@@ -0,0 +1,168 @@
+    desc: ShaderModuleDescriptor<'_>,
+) {
+    let pixel_bytes = format.target_pixel_byte_cost().unwrap();
+    let size = wgpu::Extent3d {
+        width: wgt::COPY_BYTES_PER_ROW_ALIGNMENT,
+        height: wgt::COPY_BYTES_PER_ROW_ALIGNMENT,
+        depth_or_array_layers: 1,
+    };
+    let bind_group_layout_entries = vec![wgpu::BindGroupLayoutEntry {
+        binding: 0,
+        visibility: wgpu::ShaderStages::COMPUTE,
+        ty: wgpu::BindingType::StorageTexture {
+            access: wgpu::StorageTextureAccess::Atomic,
+            format,
+            view_dimension: wgpu::TextureViewDimension::D2,
+        },
+        count: None,
+    }];
+
+    let bind_group_layout = ctx
+        .device
+        .create_bind_group_layout(&wgpu::BindGroupLayoutDescriptor {
+            label: None,
+            entries: &bind_group_layout_entries,
+        });
+
+    let pipeline_layout = ctx
+        .device
+        .create_pipeline_layout(&wgpu::PipelineLayoutDescriptor {
+            label: None,
+            bind_group_layouts: &[&bind_group_layout],
+            push_constant_ranges: &[],
+        });
+    let shader = ctx.device.create_shader_module(desc);
+    let pipeline = ctx
+        .device
+        .create_compute_pipeline(&wgpu::ComputePipelineDescriptor {
+            label: Some("image atomics pipeline"),
+            layout: Some(&pipeline_layout),
+            module: &shader,
+            entry_point: Some("cs_main"),
+            compilation_options: wgpu::PipelineCompilationOptions::default(),
+            cache: None,
+        });
+
+    let tex = ctx.device.create_texture(&wgpu::TextureDescriptor {
+        label: None,
+        dimension: wgpu::TextureDimension::D2,
+        size,
+        format,
+        usage: wgpu::TextureUsages::STORAGE_BINDING
+            | wgpu::TextureUsages::STORAGE_ATOMIC
+            | wgpu::TextureUsages::COPY_SRC,
+        mip_level_count: 1,
+        sample_count: 1,
+        view_formats: &[],
+    });
+    let view = tex.create_view(&wgpu::TextureViewDescriptor {
+        format: Some(format),
+        aspect: wgpu::TextureAspect::All,
+        ..wgpu::TextureViewDescriptor::default()
+    });
+    let bind_group = ctx.device.create_bind_group(&wgpu::BindGroupDescriptor {
+        label: None,
+        layout: &pipeline.get_bind_group_layout(0),
+        entries: &[wgpu::BindGroupEntry {
+            binding: 0,
+            resource: wgpu::BindingResource::TextureView(&view),
+        }],
+    });
+
+    let mut encoder = ctx
+        .device
+        .create_command_encoder(&wgpu::CommandEncoderDescriptor::default());
+    let mut rpass = encoder.begin_compute_pass(&wgpu::ComputePassDescriptor {
+        label: None,
+        timestamp_writes: None,
+    });
+    rpass.set_pipeline(&pipeline);
+    rpass.set_bind_group(0, Some(&bind_group), &[]);
+    rpass.dispatch_workgroups(size.width, size.height, 1);
+    drop(rpass);
+    ctx.queue.submit(Some(encoder.finish()));
```

Comment:
```suggestion
    ctx.queue.submit([encoder.finish()]);
```

Reply:
Whats the point of this change? Almost every single other file i see does the Some variant: 37 instances of Some variant and only 8 instances of [] variant.

Reply:
To try to slowly nudge people over to the [] variant - I probably should bulk change them at some point.